tg_bot.py

```diff
@@ -14,26 +16,28 @@
                     level=logging.INFO)
 logger = logging.getLogger(__name__)
 
-"""
-def nawab_twitter_authenticate():
-    auth = tweepy.OAuthHandler(config.consumer_key, config.consumer_secret)
-    auth.set_access_token(config.access_token_key, config.access_token_secret)
-    api = tweepy.API(auth)
-    return api
-"""
+auth = tweepy.OAuthHandler(config.consumer_key, config.consumer_secret)
+auth.set_access_token(config.access_token_key, config.access_token_secret)
+api = tweepy.API(auth)
```

Comment:
This is not the correct way to do this. Either call from the functions we have in a twitter bot script or you can try to have a single interface of interaction.

Reply:
Also, the code is written globally. Which is not a good practice either.